versions/3.1.1.md

```diff
@@ -967,7 +979,7 @@ Describes a single API operation on a path.
 | <a name="operation-callbacks"></a>callbacks       | Map[`string`, [Callback Object](#callback-object) \| [Reference Object](#reference-object)] | A map of possible out-of band callbacks related to the parent operation. The key is a unique identifier for the Callback Object. Each value in the map is a [Callback Object](#callback-object) that describes a request that may be initiated by the API provider and the expected responses.                                                                                                                                                                                                                                                                                                             |
 | <a name="operation-deprecated"></a>deprecated     |                                         `boolean`                                         | Declares this operation to be deprecated. Consumers SHOULD refrain from usage of the declared operation. Default value is `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
-| <a name="operation-security"></a>security         |                [[Security Requirement Object](#security-requirement-object)]                | A declaration of which security mechanisms can be used for this operation. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. To make security optional, an empty security requirement (`{}`) can be included in the array. This definition overrides any declared top-level [`security`](#oas-security). To remove a top-level security declaration, an empty array can be used.                                                                                            |
+| <a name="operation-security"></a>security         |                [[Security Requirement Object](#security-requirement-object)]                | A declaration of which security mechanisms can be used for this operation. If present, then this overrides any [top-level `security` field](#oas-security) for this operation. See the [definition of that field](#the-security-field) for details.
```

Comment:
```suggestion
| <a name="operation-security"></a>security         |                [[Security Requirement Object](#security-requirement-object)]                | A declaration of which security mechanisms can be used for this operation. If present, then this overrides any [top-level `security` field](#oas-security). See the [definition of that field](#the-security-field) for details.
```

@ralfhandl WDYT? The pre-suggestion text could be read as "there's a top-level security field, just for this operation", which is confusing and doesn't make sense.

Reply:
Yes, that is clearer. An alternative would be
```suggestion
| <a name="operation-security"></a>security         |                [[Security Requirement Object](#security-requirement-object)]                | A declaration of which security mechanisms can be used for this operation. If present, this overrides a `security` field specified at the [OpenAPI Object](#oas-security) level. See the [definition of that field](#the-security-field) for details.
```

Reply:
Yeah, except that reads strangely in the case where there _is_ no top-level `security`.